src/custom/hooks/useParseReferralQueryParam.ts

```diff
@@ -1,7 +1,8 @@
-import { useMemo } from 'react'
+import { useMemo, useState } from 'react'
 import useParsedQueryString from 'hooks/useParsedQueryString'
 import { REFERRAL_QUERY_PARAM } from 'hooks/useReferralLink'
 import useENS from 'hooks/useENS'
+import { isAddress } from '../utils'
```

Comment:
Can you use the full path here? I think it should be

```suggestion
import { isAddress } from 'hook/utils'
```

Reply:
I guess this was not the right path... sorry!

![Screen Shot 2022-04-18 at 11 08 44](https://user-images.githubusercontent.com/43217/163793452-58881f08-5d25-4873-83a6-706ea1c3a173.png)

Will need to figure out the correct one

Reply:
no worries! just fixed it